circe/decoupled_pricing/compute.py

```diff
@@ -509,22 +540,34 @@ def process_IN_CLOSE_WRITE(self, event):
                     runtime_info = 'rt_finish '+ input_name + ' '+str(ts)
                     send_runtime_profile_computingnode(runtime_info,task_name,home_id)
 
-                    for idx,host in enumerate(next_hosts):
-                        current_file = files_mul[key][idx].split('/')[-1]
-                        destinations = "/centralized_scheduler/input/" +next_tasks_map[task_name][idx]+"/"+home_id+"/"+current_file
-                        if self_ip!=combined_ip_map[host]:
-                            transfer_data(host,username,password,files_mul[key][idx], destinations)
-                        else: 
-                            copyfile(files_mul[key][idx],destinations)
-
+                    # Using unicast 
+                    # for idx,host in enumerate(next_hosts):
+                    #     current_file = files_mul[key][idx].split('/')[-1]
+                    #     destinations = "/centralized_scheduler/input/" +next_tasks_map[task_name][idx]+"/"+home_id+"/"+current_file
+                    #     if self_ip!=combined_ip_map[host]:
+                    #         transfer_data(host,username,password,files_mul[key][idx], destinations)
+                    #     else: 
+                    #         copyfile(files_mul[key][idx],destinations)
+
+                    # Using multicast
+                    destinations = []
+                    logging.debug('Using multicast instead')
+                    for idx in range(0,len(next_hosts)):
+                        current_file = files_mul[key][idx].split('/')[-1]  
+                        cur_des =  "/centralized_scheduler/input/" +next_tasks_map[task_name][idx]+"/"+home_id+"/"+current_file 
```

Comment:
Limit lines to 80 columns. I like to have 2-4 code files open on a single screen at times to read code. If lines are beyond 80 columns, its really hard to read.

Reply:
sure, thanks!